lib/order.rb

```diff
@@ -0,0 +1,54 @@
+require './lib/send_sms'
+
+class Order
+  include Message
```

Comment:
Modules are generally used when you want to include extra functionality in a class. For example, now that you've included `Message`, `Order` has access to `@to`, `@from`, `@account_sid` etc.

Is this what you want? Does an Order need access to these variables? If not, you probably don't need to use a Module here, and are better off creating a Message class that can encapsulate this data.

Modules are generally used when you want to share a function between multiple classes.